packages/wct-mocha/src/reporters/multi.ts

```diff
@@ -117,6 +118,9 @@ export default class MultiReporter implements Reporter {
     this.emit('end');
   }
 
+  epilogue(): void {
```

Comment:
what's epilogue?

Reply:
Sounds neat, right?  I don't 100% know.  Here's someone talking about it https://github.com/mochajs/mocha/issues/3200#issuecomment-356977628 but I added it because typescript said I didn't actually implement a Reporter unless I had one.